gozw.go

```diff
@@ -325,6 +323,33 @@ func (c *Client) AddNode() (*Node, error) {
 		}
 	}
 
+	progress <- PairingProgressUpdate{
+		InterviewedCommandClassCount: 0,
+		ReportedCommandClassCount:    len(node.CommandClasses),
+	}
+
+	c.l.Debug("reported command classes", zap.Int("len", len(newNodeInfo.CommandClasses)))
+
+
+	go func() {
+		lastReportedInterviewLength := 0
+		for {
+			currentProgress := node.InterviewedCommandClassCount
+			if currentProgress != lastReportedInterviewLength {
```

Comment:
nit: you could check if currentProgress == lastReportedInterviewLength and continue, so that you don't have to nest logic as much